### PR TITLE
Bundles

### PR DESCRIPTION
**Pull request by SunkyMP3G**
Why do I need to make 2 pull requests for news and bundles?

### CODE REVIEW

**Comment by JojoFR1** [comment]
> Why do I need to make 2 pull requests for news and bundles?

Yes, why. And why do people remove the nice Pull Request Template?

**Comment by SunkyMP3G** [comment]
> Yes, why. And why do people remove the nice Pull Request Template?
I remove it only in translations, because it's useless

**Review by JojoFR1 — CHANGES_REQUESTED** [review]

File: src/assets/bundles/bundle_ru.properties [hunk 1]
```diff
@@ -295,26 +295,26 @@ urlNews = https://raw.githubusercontent.com/FredyJabe/aeyama/main/src/assets/new
...
+objective.mission.new-world07 = Теперь исследуйте сборщик древесины, чтобы вы могли собирать доски, не глядя.
+objective.mission.new-world08 = Хорошо, теперь разместите их возле деревьев.
+objective.mission.new-world09 = Теперь соберите ещё 50 камня, нам его понадобится много.
 objective.mission.new-world10 = Отлично, теперь изучите некоторые базовые вещи, чтобы начать работу с небольшой базой!
-objective.mission.new-world11 = You're doing a great job so far, build some Stone Miner to really get us going.
+objective.mission.new-world11 = Вы делаете успехи, постройте каменные буры на камне.
 objective.mission.new-world12 = Наши сканеры засекли в этом районе источник железа,\nисследуйте то, что вам нужно, чтобы добыть его и получить.
-objective.mission.new-world13 = Perfect! Go to the location on your map, build some miners and collect 50 raw iron.
-objective.mission.new-world14 = Nice, everything going all good. Now go back to your core!
-objective.mission.new-world15 = You'll need to research and build\nstockpiles beside that Drop Pod of yours to increase\nyour resource capacity!
```
[the remark this answers is not in the document]
Comment (JojoFR1) [reply src/assets/bundles/bundle_ru.properties:313]: *Just noticed I had to request change, don't know how it works eh*

**Comment by SunkyMP3G** [comment]
Fixed

**Pull request merged**